### ISSUE TITLE
Serialize 3 tag types to YAML

### ISSUE THREAD

**Issue by Mimickal** [Mimickal/Librarian#3]
Pick 3 tag types to serialize to YAML. I don't care which ones are picked, but it'd be helpful if they were basic. Hopefully in doing 3 of these, we can identify some patterns for making this process a little more dynamic so we don't have to tediously add every current tag type or new tag type that may come along.

If we do identify a pattern, *don't* implement it in this task. Create a new task with your findings and we'll move from there.

**Comment by Mimickal** [comment]
Blocked on #1

**Comment by ShellyHerself** [comment]
| type | challange
| --- | :--: |
| actor | basic structure |
| bitmap |  raw data |
| shader_transparent_chicago |  arrays and inheritance |

**Comment by Mimickal** [comment]
I do want to avoid anything complicated for now. I may rewrite this task to just be a spike for how we can make this data-driven or something else, if we even can. Not sure yet, I need to do a little work on it and see how things shake out.
